.github/workflows/ci-notebooks.yml

```diff
@@ -0,0 +1,29 @@
+name: ci-notebooks
+on:
+  pull_request:
+    paths:
+      - modin/**
```

Comment:
This is done to track modin changes which potentially can break notebooks as it was proposed in https://github.com/modin-project/modin/pull/4145#discussion_r801597095. Otherwise notebooks can be broken and we won't know this until someone will change notebooks code, which triggers this PR tests execution.

Reply:
That probably makes sense.